examples/base_cc_gwlb_zpa/terraform.tfvars

```diff
@@ -144,26 +144,39 @@
 
 #cross_zone_lb_enabled                      = true
 
+## 15. Gateway loadbalancing hashing algorith. Zscaler recommended default is 2-tuple (source_ip_dest_ip).
```

Comment:
algorith->algorithm

Reply:
fixed in commit 13cda2542b0e1ecb28a8e65b6f59cd5309b13dd5